client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/useInteractionsTab.ts

```diff
@@ -90,17 +88,17 @@ const useInteractionsTab = (parameters: useInteractionsTabParameters): useIntera
                 }
             }
             return previous;
-        }, {familyMembers: [], educationMembers: []});
+        }, { familyMembers: [], educationMembers: [] });
     }
 
     const loadInvolvedContacts = () => {
         const loadInvolvedContactsLogger = logger.setup('loading involved contacts');
         loadInvolvedContactsLogger.info('launching db request', Severity.LOW);
-        axios.get(`/intersections/involvedContacts/${epidemiologyNumber}`)
-        .then((result) => {
+
+        axios.get(`/intersections/involvedContacts/${epidemiologyNumber}`).then((result) => {
             if (result?.data && result.headers['content-type'].includes('application/json')) {
                 loadInvolvedContactsLogger.info('got response successfully', Severity.LOW);
-                const involvedContacts : InvolvedContact[] = result?.data;
+                const involvedContacts: InvolvedContact[] = result?.data;
```

Comment:
Don’t you need to update the involved contact interface with the new types?

Reply:
No, I used the same types